root_data.tf

```diff
@@ -29,3 +29,15 @@ data "aws_ssm_parameter" "workflow_pat" {
 data "aws_ssm_parameter" "slack_webhook_url" {
   name = "/mgmt/release/slack/webhook"
 }
+
+data "aws_ssm_parameter" "gpg_passphrase" {
```

Comment:
Is there a reason for the whitespace in the names of the params?

Reply:
I'm just not capable of copy and paste.